tests/test-team/config/auth.setup.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Bit of a weird one

When running the web-app in dev mode `npm run dev` then running ALL the automated tests. The component tests become flakey.

Running the web-app in production mode `npm run buld && npm run start` the tests pass quickly and are not flakey.